src/plugins/data/common/field_formats/converters/_numeral.ts

```diff
@@ -22,11 +22,7 @@ import numeral from '@elastic/numeral';
 // @ts-ignore
 import numeralLanguages from '@elastic/numeral/languages';
 import { assign, has } from 'lodash';
-import {
-  FieldFormat,
-  KBN_FIELD_TYPES,
-  TextContextTypeConvert,
-} from '../../../../../../plugins/data/common/';
+import { FieldFormat, KBN_FIELD_TYPES, TextContextTypeConvert } from '../../index';
```

Comment:
I can recommend you to avoid of using import from parent index file. It can be a reason of cross references in future

Reply:
Done